Portfolio

Najeeba-Qarqin opened the pull request:
Portfolio project created in HTML, CSS and JavaScript and designed by UI/UX designers.

<img width="960" alt="Screenshot 2024-08-04 010547" src="https://github.com/user-attachments/assets/9567029a-aeaf-4347-8e1a-24d512f5ee90">

Najeeba-Qarqin committed:
Added lintes file and images

Najeeba-Qarqin committed:
Added images

Najeeba-Qarqin committed:
Created html file

Najeeba-Qarqin committed:
Added starting codes of html file

Najeeba-Qarqin committed:
Added header and home section

Najeeba-Qarqin committed:
Added js file and code for functionality of huburger menu btn

Najeeba-Qarqin committed:
Added code for highlite the navbar links when window scrolled

Najeeba-Qarqin committed:
Added skills and projects sections

Najeeba-Qarqin committed:
Added dynamic contents to home and about sections

Najeeba-Qarqin committed:
Added content of skills

Najeeba-Qarqin committed:
Added content of projects section

Najeeba-Qarqin committed:
Looped the content of projects card and popup into html file

Najeeba-Qarqin committed:
Added contact section and contact form in html file

Najeeba-Qarqin committed:
Changed the double qouts to single quotes

Najeeba-Qarqin committed:
Added validation code for form and added code for storing the information in local storage

Najeeba-Qarqin committed:
Created css file and imported google font

Najeeba-Qarqin committed:
Added style for root element, universal elements and header

Najeeba-Qarqin committed:
Added style for logo toggle btn and navbar section

Najeeba-Qarqin committed:
Added style to home section

Najeeba-Qarqin committed:
Added style to about section

Najeeba-Qarqin committed:
Added style to projects cards and popup sections

Najeeba-Qarqin committed:
Added style for skills section

Najeeba-Qarqin committed:
Added style for contact section

Najeeba-Qarqin committed:
Added style for tablate version

Najeeba-Qarqin committed:
Added styles for laptop version

Najeeba-Qarqin committed:
Solved linter errors

Najeeba-Qarqin committed:
Solved linter error

Najeeba-Qarqin committed:
Cleaned html code

Najeeba-Qarqin committed:
Solved linter errors

Najeeba-Qarqin committed:
Solved linter error

Najeeba-Qarqin committed:
Solved linter error

Najeeba-Qarqin committed:
Solved linter error

Najeeba-Qarqin committed:
Solved linter error

Najeeba-Qarqin committed:
Solved linter error

Najeeba-Qarqin committed:
Solved linter error

Najeeba-Qarqin committed:
Solved linter error

Najeeba-Qarqin committed:
Solved linter error

Najeeba-Qarqin committed:
Solved linter error

Najeeba-Qarqin committed:
Added description to readme file

Najeeba-Qarqin commented:
Hi @Hasinarahman👋 
Please review my project.

Najeeba-Qarqin committed:
Changed image name

Hasinarahman commented:
Hi @Najeeba-Qarqin  🙋‍♀️,
Highlights 🌞
🎉 Great Design Implemented
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed the Personal Portfolio project equirments
✔️ You have followed the all design of figma from UX/UI student
✔️ all of your links works good
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
✔️Your code is clean
✔️Your repo has a README.md file

just one thing to say your view button of the certificate is so close to the another certificate please give some space on design it.

Najeeba-Qarqin commented:
Hi @Hasinarahman 
Thank you for your feedback!
I'll implement them.

Najeeba-Qarqin committed:
Added margin for certificates

Najeeba-Qarqin commented:
@Hasinarahman I implemented your feedback please review again.

ahzamir commented:
@Najeeba-Qarqin 

- Well done Najeeba, your project is almost done but has some minor issues first of all before I check your project design and functionality, I need to match it with the Figma design so please provide the link to the Figma design in reply to this message and also on your README.md file.

ahzamir commented on script.js:
```diff
@@ -0,0 +1,251 @@
+  tools: ['Git', 'GitHUb'],
+  certification: [
+    { name: 'Web Design', link: 'www.com', image: './images/webDesign.png' },
+    { name: 'JavaScript', link: 'www.com', image: './images/JS.png' },
```
- Add the actual link to your certifications.

ahzamir commented on script.js:
```diff
@@ -0,0 +1,251 @@
+    longDescription: 'Portfolio project a simple personal portfolio website one of certification projects in freeCodeCamp Responsive Web Design program build in HTML and designed in CSS.',
+    image: './images/Portfolio.png',
+    technology: ['HTML', 'CSS', 'Git', 'GitHub'],
+    liveLink: 'https://najeeba-qarqin.github.io/Prtfolio',
```
- This link is broken and not working

ahzamir commented on script.js:
```diff
@@ -0,0 +1,251 @@
+    longDescription: 'Pokemon Search App project a simple app can show data and information about any pokemon from their ID or name, one of certification projects in freeCodeCamp Algorithms And Data Structure program build in HTML, CSS and JavaScript.',
+    image: './images/PokemonSearch.png',
+    technology: ['HTML', 'CSS', 'Git', 'GitHub'],
+    liveLink: 'https://najeeba-qarqin.github.io/Prtfolio',
```
- This link is broken, please add the actual live link for this project.

ahzamir commented on index.html:
```diff
@@ -0,0 +1,170 @@
+      <p class="para">If you have an application you are interested in developing, a feature that you need builr or a
+        project that need coding. I'd love to help with it. Or if you have any feedback and message please share with
+        me.</p>
+      <form action="" name="myform" class="myform flex" id="myform" method="get">
```
- You have to use [Formspree](https://formspree.io/) for you contact form submission action.
  - You can check the project's requirements and the documentation of the formspree for more details.

ahzamir commented on script.js:
```diff
@@ -0,0 +1,251 @@
+const emailError = document.getElementById('email_error');
+const messageError = document.getElementById('message-error');
+form.addEventListener('submit', (e) => {
+  const emailChecker = /^[\w-]+@[a-zA-Z\d-]+\.[a-zA-Z]{2,}$/;
```
- You have to check if the user writes his email in capital or small letters and if it has any capital letters you have to show an error message:
  - Check the project requirements for more details.

ahzamir commented on README.md:
```diff
@@ -1 +1,122 @@
+Examples:
+
+- I learned how to make dynamic webpage using js.
+- You can give some information about me.
```
- Solve this, as the user does not give information about you they will read info about you.

ahzamir commented on README.md:
````diff
@@ -1 +1,122 @@
+}
+
+nav li a {
+  text-decoration-line: none;
+  font-size: 10px;
+  font-weight: 600;
+}
+
+
+```
+
+#### JS code sample
+
+
+```js
+
+
+function showPopup(index) {
+  const project = projects[index];
+  popupContent.innerHTML = `
+  <h3 class='span'>${project.title}</h3>
+  <div class='project-popup flex'>
+    <img class="projects-images" id='popup-image' src='${project.image}' alt='project-img' />
+    <p class='para'>${project.longDescription}</p>
+    </div>
+    <h2 class='para'><span class='span'>Technologies</span>: ${project.technology.join(', ')}</h2>
+    <div id='popup-btns'>
+    <a class='button' href='${project.liveLink}' target='_blank'>Live Demo</a>
+    <a class='button' href='${project.sourceLink}' target='_blank'>Source</a>
+    </div>
+  `;
+  popup.classList.remove('hidden');
+};
+
+popupBtn.addEventListener('click', () => {
+  popup.classList.add('hidden');
+});
+
+
+```
````
- You can remove this part.

Najeeba-Qarqin replied:
Why sir, and how??🥺

Najeeba-Qarqin replied:
Understand 🙂

ahzamir commented on README.md:
```diff
@@ -1 +1,122 @@
+## Demo 📸
+
+
+![Demo](/images/DEMO.png)
```
- The demo image is not showing up as your image is `Demo.png` not `DEMO.png`

ahzamir commented on README.md:
```diff
@@ -1 +1,122 @@
+
+### Live Link
+
+[The_live_link_of_this_project](https://najeeba-qarqin.github.io/Portfolio-JS/)
```
- This link is not working, but no worries you can update it after you deployed this project

ahzamir commented on README.md:
```diff
@@ -1 +1,122 @@
+
+[The_live_link_of_this_project](https://najeeba-qarqin.github.io/Portfolio-JS/)
+
+## Technologies Used 🛠️
+
+
+- HTML
+- CSS
+- JS
+
```
- Change this to Technologies and Tools used and then also add Figma on the below list

ahzamir commented on README.md:
```diff
@@ -1 +1,122 @@
+- Email: [Najeeba Qarqin](najeebaqarqin@gmail.com)
+- GitHub: [Najeeba Qarqin](https://github.com/Najeeba-Qarqin)
+- UI/UX Designers:
+  ####  Maryam Yousufi
+  ####  Sara Shirzad
+  ####  Sadaf Ahmadi
+  ####  Farkhunda Raufi
```
- Add a link to the Figma design here and also you need to add at least one social media link for these UI/UX designers.

ahzamir reviewed (changes requested):
Hey @Najeeba-Qarqin , Well done and great job by implementing most of the requirements:

- Well done your project's functionality almost meets most of the requirements, but still your project has some minor issues you need to work on.

- This review and feedback is just for your project's README.md, HTML, and JavaScript requirements. In order to get feedback and review for your design you have to provide link of the Figma design on your readme file.

Najeeba-Qarqin commented:
> @Najeeba-Qarqin
> 
> * Well done Najeeba, your project is almost done but has some minor issues first of all before I check your project design and functionality, I need to match it with the Figma design so please provide the link to the Figma design in reply to this message and also on your README.md file.

@ahzamir  Hi sir thank you for feedback 
Ok this is the link: 
 https://www.figma.com/invites/claim/M4WghDpaGEamietPGHcEHoCfP4GsHkX0yXRl13g9J5TR?fuid=1379424393568974635

Najeeba-Qarqin committed:
Added action for form in html file changed the links of certificate and projects in ja file and removed code samples changed demo and technologies used added figma design link and social media links of figma designer

Najeeba-Qarqin committed:
Added code to check the email is upper case

Najeeba-Qarqin committed:
Fixed closing curly brase issue

Najeeba-Qarqin commented:
@ahzamir Hi sir!
I implemented  your feedback can you please check the project again.

Najeeba-Qarqin committed:
Added likedin link for readme author section

ahzamir commented on script.js:
```diff
@@ -0,0 +1,255 @@
+  } else {
+    messageError.innerHTML = '';
+  }
+});
```
@Najeeba-Qarqin 

- Well done, just one thing else you need to implement here after the user submits the form you need to clear the Form fields inner text and the LocalStorage Data.

ahzamir commented on styles.css:
- The background color for the project pages including the project header is a linear gradient color, you have to use the exact linear gradient color on your project background, please have a look at the images to see the changes between your design and the Figma design:
- The buttons' background color also has little gradient and shadow on the Figma Design.
- The padding and sizing of the Elements on the Home page are a little bit different from the Figma design

- The Figma Design:
![image](https://github.com/user-attachments/assets/19ae6db0-c043-4e53-9b1f-801a7e27073a)
|
![image](https://github.com/user-attachments/assets/0a0518bf-c9b8-43fc-b921-c80956bbb122)


- Your Design:
  - Desktop version:
  - Here I added just the desktop version design as a reference but implement these changes for both desktop and mobile versions.
  
![image](https://github.com/user-attachments/assets/4f53470e-8169-4f77-9f0a-456dfca04ce5)
|
![image](https://github.com/user-attachments/assets/24965ab4-a0bf-4383-95f8-bfeb1e660afc)

Najeeba-Qarqin replied:
> * The background color for the project pages including the project header is a linear gradient color, you have to use the exact linear gradient color on your project background, please have a look at the images to see the changes between your design and the Figma design:
> * The buttons' background color also has little gradient and shadow on the Figma Design.
> * The padding and sizing of the Elements on the Home page are a little bit different from the Figma design
> * The Figma Design:
>   ![image](https://private-user-images.githubusercontent.com/96838030/353931129-19ae6db0-c043-4e53-9b1f-801a7e27073a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjI0NTkxMzcsIm5iZiI6MTcyMjQ1ODgzNywicGF0aCI6Ii85NjgzODAzMC8zNTM5MzExMjktMTlhZTZkYjAtYzA0My00ZTUzLTliMWYtODAxYTdlMjcwNzNhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MzElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzMxVDIwNDcxN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWRiNjQzMTQxYmM5ZTQ0OWFhMDI2NTEwOTAxMGNlZjBiNjdhMzcxNGI5YzVlMThmN2ZiYjFkZDg0MWYyOGQ0YmImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.1jq7C_ZESZNsnrgNPbl1iv8Dwd4_V_BuG3o78NJ7P_A)
>   |
>   ![image](https://private-user-images.githubusercontent.com/96838030/353932378-0a0518bf-c9b8-43fc-b921-c80956bbb122.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjI0NTkxMzcsIm5iZiI6MTcyMjQ1ODgzNywicGF0aCI6Ii85NjgzODAzMC8zNTM5MzIzNzgtMGEwNTE4YmYtYzliOC00M2ZjLWI5MjEtYzgwOTU2YmJiMTIyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MzElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzMxVDIwNDcxN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWVkOThiZTZkZmYzNzBhZDgwNTI1NTM4MTI3YjZkYWVlNzZkNmM0MWE2N2I5MGRlZjY1MWRkYmQ1MjAyYWUzN2EmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.M4Imu6r8BG3XQbPxMysNxdzFnBGUEbXpRkV1_f2x6bk)
> * Your Design:
>   
>   * Desktop version:
>   * Here I added just the desktop version design as a reference but implement these changes for both desktop and mobile versions.
> 
> ![image](https://private-user-images.githubusercontent.com/96838030/353933085-4f53470e-8169-4f77-9f0a-456dfca04ce5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjI0NTkxMzcsIm5iZiI6MTcyMjQ1ODgzNywicGF0aCI6Ii85NjgzODAzMC8zNTM5MzMwODUtNGY1MzQ3MGUtODE2OS00Zjc3LTlmMGEtNDU2ZGZjYTA0Y2U1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MzElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzMxVDIwNDcxN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTBhZjExOWI4NTBhNzdhYzBjODc3Mzg4YjA0ZDQyZGVjMjI0N2NmMjI1OGE2ZjlmODY0ZDEzN2Q4Mzc2YmM0NzkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.wqWigpA1kpS9oI1rtRANCex3d5BEVzXkpiNVuBPKirs) | ![image](https://private-user-images.githubusercontent.com/96838030/353932217-24965ab4-a0bf-4383-95f8-bfeb1e660afc.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjI0NTkxMzcsIm5iZiI6MTcyMjQ1ODgzNywicGF0aCI6Ii85NjgzODAzMC8zNTM5MzIyMTctMjQ5NjVhYjQtYTBiZi00MzgzLTk1ZjgtYmZlYjFlNjYwYWZjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MzElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzMxVDIwNDcxN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTJkZjllZDA1MTVkYjc4ODNjYzdkOGI3NWY4MTczODZjYmNhNWYzZDA3YzQ0ZGY0ZTliZDYzMTZiYmFhMTk3NWEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Fbd92tXnd2o57MjCXe80Wy3GXpfU66dsDdTYDWNsTSg)

@ahzamir 
in the body the linear gradient color is for it's border sir not for background i looked at it have only one color for background if i add the linear gradient it make sense like light color.

ahzamir replied:
- From my point of view, I think the body background is a linear gradient consisting of 3 colors, check the Figma design there you can find which colors with which percentages!

ahzamir commented on styles.css:
- The placing od the button on the about page is different from the Figma design:
- Same issue on the About page, the button design and sizing are not aligned and the same as the Figma design:

- Your Design:
![image](https://github.com/user-attachments/assets/5a7f6974-fae6-4935-8578-d1177bd7c9b3)

![image](https://github.com/user-attachments/assets/96df1114-31f5-4c3a-a78d-40393b0a7cb0)

- The Figma Design:
![image](https://github.com/user-attachments/assets/f1dc2bf9-90bb-4060-8445-f4dd53fa0fbc)
![image](https://github.com/user-attachments/assets/9aed580c-2889-4c6a-a57d-d4c1ad9b552e)

ahzamir commented on styles.css:
- Padding issue on the skills page, please check the Figma design and add padding to your elements according to the Figma Design:
- Also the text under skills should be centered

- Your Design Mobile version:
![image](https://github.com/user-attachments/assets/9a090a67-2d41-4bc0-9721-c2a0843c75f2)

- Figma Design:
![image](https://github.com/user-attachments/assets/8a2c95c7-02f3-4744-8044-1f0f368c86e4)

ahzamir commented on styles.css:
@Najeeba-Qarqin 

- Although I accept that the changes you added is looking more professional and more cool on your project, but as this project is a lesson for you to follow the exact design which is on the Figma so you need to design exactly like the Figma Design:
- No background for the projects card:

- Your design:
![image](https://github.com/user-attachments/assets/3f5751d0-68bf-4dd9-864f-b7b3cab50b1a)

- Figma Design:
![image](https://github.com/user-attachments/assets/fa770660-220b-4f95-83cc-0304f5d8cc48)

ahzamir commented on styles.css:
- These button icons imported with a very low format graphic although it's correct on the Figma solve this:
![image](https://github.com/user-attachments/assets/02d29d4e-a592-4a6a-8b8a-1f021a136032)

ahzamir reviewed (changes requested):
@Najeeba-Qarqin 

- Well done Najeeba by implementing the feedback and creating this amazing project just some minor changes on the design of the project are remaining you need to work on:

Najeeba-Qarqin committed:
Reseted the form and cleared the local storage when form submited and changed the sizing and padding in the home page changed the background color of buttons removed the background of projects card

Najeeba-Qarqin committed:
Solved linter errors

Najeeba-Qarqin commented:
@ahzamir Salam sir!
I changed the required points of design that you mentioned just about the background of the body please check again cause when i added linear gradient it makes big difference.

ahzamir commented:
> @ahzamir Salam sir! I changed the required points of design that you mentioned just about the background of the body please check again cause when i added linear gradient it makes big difference.

I checked it, it's a linear gradient background.

Najeeba-Qarqin commented:
> > @ahzamir Salam sir! I changed the required points of design that you mentioned just about the background of the body please check again cause when i added linear gradient it makes big difference.
> 
> I checked it, it's a linear gradient background.

![Screenshot_20240801-195922_Chrome](https://github.com/user-attachments/assets/2eb6dac6-f7e8-4ffd-941f-39273431142f)
Sir can you explain please it's just one color for background 🥺

Najeeba-Qarqin committed:
Added radial gradient to background

Najeeba-Qarqin committed:
Solved linter error

Najeeba-Qarqin committed:
Removed background of nav

Najeeba-Qarqin commented:
@ahzamir 
Hi sir!
I added a radial gradient color for background of project.

Najeeba-Qarqin committed:
Changed the demo image

ahzamir reviewed (approved):
### Hi @Najeeba-Qarqin 🙋‍♀️,

> Excellent work! 💯 You have done a great job implementing all the project requirements. 🥳 Your feature is complete! Congratulations!!!🎊

- **[REQUIRED]** Share your project on this [GROUP](https://www.facebook.com/groups/kick-startdev), add a simple Proud Moment text and a screenshot of your project then besides that add your project link at the end of the text as a GitHub Source code reference and post.

- Share your Project on your LinkedIn account and add a proud moment text with a screenshot.

- Well Done great design Implemented. 🎯🎉👍
![image](https://github.com/user-attachments/assets/13ae81e2-ff65-417d-ba1f-cdb7219e816a)

# Status: Approved ✅
There is nothing to say other than to merge your branch. 🎉🎯✅👍😊🥳

![WinkThumbsUpGIF](https://github.com/ZohalMirkhel/palindrome/assets/96838030/e5ee293e-072a-4806-bde0-b64e13140efa)


 ## Highlights 🌞

🎉 Great Design Implemented 
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)✔️ You have followed the Cash Register project task and committed using the command line
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
✔️Your code is clean
✔️Your repo has a professional README.md file
✔️ Your partner has reviewed your repo
✔️Your design is responsive
✔️Your design is according to the Figma design and even better than that. 🎯

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me @ahzamir in your question so I can receive the notification.**

ahzamir commented:
@Najeeba-Qarqin 

- Don't forget to:
  - Merge
  - Deploy
  - Share on FB Group & Social Media

> These are part of the requirments
